### PR TITLE
Adding tracking information in the pixel client

### PR DESCRIPTION
**Pull request by akhatiwa**
Adding tracking information merging special client pixel_timingScan_dqm_sourceclient-live_cfg.py with pixel_dqm_sourceclient-live_cfg.py. Needed to modify SiPixelPhase1OnlineDQM_Timing_cff.py for the same purpose. CMSSW release: CMSSW_10_0_2.

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

File: DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 process = cms.Process("PIXELDQMLIVE", eras.Run2_2017)
 
-live=True  #set to false for lxplus offline testing
```
Comment (fioriNTU) [inline DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py]: @akhatiwa live needs to be set at true, please fix it and push again (also in the 10_1_X PR)

File: DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py [hunk 2]
```diff
@@ -78,18 +78,19 @@
 elif(offlineTesting):
     process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
     from Configuration.AlCa.GlobalTag import GlobalTag as gtCustomise
-    process.GlobalTag = gtCustomise(process.GlobalTag, 'auto:run2_data', '')
+    process.GlobalTag = gtCustomise(process.GlobalTag, '100X_dataRun2_v1', '')
```
Comment (fioriNTU) [inline DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py]: please leave auto:run2_data as global tag in the code, it is more generic and can generally be used with any CMSSW release (tha same apply to the 10_1_X PR)

**Comment by akhatiwa** [comment]
Hi @vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien, 

Could you please check and sign the pull request again. Thank you.

**Comment by dmitrijus** [comment]
+1

**Comment by dmitrijus** [comment]
backport of https://github.com/cms-sw/cmssw/pull/22336

**Comment by fioriNTU** [comment]
Please @akhatiwa close this PR, it introduce a crash in cosmics, thanks.

**Pull request closed without merging**